content/blog/cc-product-roadmap-1/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Product Roadmap
+date: 2019-11-22
+tags: 
+  - product
+  - plans
+  - announcements
+image: release-1.jpg
+author: crysfel-villa
+---
+Over the last couple weeks we've been working on the new features we want to implement in the platform. We've finally came up with a plan for the improvements we will be working over the next couple months.
+
+While this is not set on stone, it'll definately give us the direction we need to keep this project moving forward. I'm really excited about the new things to come and I hope all of you can join us in this adventure.
+
+One of the goals is to deliver features as quickly as we can, for that reason we are going to release all new features incrementally. We are planning to have four small/medium releases over the next several months.
+
+### Release 1: User profiles
+At the moment mentees can't really do anything in the plaform, but it would be great if any user can update their profile so mentors can see their goals and expectations when a mentee reach out to them.
+
+For this firt release, we are going to create a new private area for members, the profile page will look like this, from here members can update their profile.
+
+![User profile](./release-1.jpg)
+
+We will allow members to update their avatar as well. For now there will be only two options in the main navigation menu, `Home` and `Mentors`.
+
+The Home link will redirect the user to the profile, later on we want to show several widgets here to provider mentors and mentees with useful information about their mentorships.
+
+The Mentors link will send users to the current app we have, from there people will be able to search for mentors.
+
+### Release 2: Mentor availability and reports
+On top of the user profile, we are going to build the mentor profile page, the design is the same but there will be more fields, as mentors can define communication channels and availability.
+
+We are also planning to build an easy way to report any issues in the mentorships by creating a new reports module. Mentors and mentees will be able to report missconducts and reach out to Coding Coach stuff.
+
+![Mentor profile](./release-2.png)
+
+### Release 3: Apply for mentorships
+One of the problems we currently have is that mentorships are happening outside the platform, a mentee will reach out to a mentor and ask for help, but then we don't know if that relationship is successfull or not.
+
+This is the largest release in the roadmap, because we need to introduce a new process to make the mentorships happen in the platform, this way we are preparing the platform for new features we want to include later on.
```

Comment:
```suggestion
This is the largest release in the roadmap because we need to introduce a new process to make the mentorships happen in the platform, this way we are preparing the platform for new features we want to include later on.
```